string junction_name
int32[] robot_ids